### ISSUE TITLE
Where is zero-shot training reflected

### ISSUE THREAD

**Issue by DonMuv** [Mwxinnn/AA-CLIP#12]
Hi, i am a researcher in other fields. I saw in the code that pixel-level masks are used as labels during model training:

loss += calculate_seg_loss(patch_preds, mask)

So where is zero-shot training reflected?

**Comment by Mwxinnn** [comment]
We use one dataset (VisA) to adapt clip and test on other datasets which are not seen during training ;)
